terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/validation/index.jsp

```diff
@@ -0,0 +1,13 @@
+<h2>Validation Function Test</h2>

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...